vendor/bat-native-ads/src/bat/ads/internal/ads/new_tab_page_ads/new_tab_page_ad_unittest.cc

```diff
@@ -125,8 +120,6 @@ TEST_F(BatAdsNewTabPageAdTest, FireViewedEvent) {
 
 TEST_F(BatAdsNewTabPageAdTest, FireClickedEvent) {
   // Arrange
-  new_tab_page_ads::frequency_capping::ForcePermissionRules();
-
```

Comment:
See previous comment concerning `This logic is still required if ads are disabled`